index.js

```diff
@@ -84,6 +84,10 @@ function _supportsColor(haveStream, {streamIsTTY, sniffFlags = true} = {}) {
 		return 0;
 	}
 
+	if (env.COLORTERM === 'truecolor') {
+		return 3;
+	}
```

Comment:
Why is this moved?

Reply:
> Why is this moved?

COLORTERM is present in most cases. And it reduces the number of times Deno needs to obtain permissions for every env it accesses. Deno has to request permission once for each access to an env.

Reply:
This technically breaks our API...

Reply:
Restored